app/assets/javascripts/common/reports.js

```diff
@@ -1094,10 +1093,8 @@ function baseLineGraphConfig() {
       scales: {
         x: {
           stacked: false,
-          display: true,
           grid: {
             display: false,
-            drawBorder: true,
```

Comment:
```scales[id].grid.drawBorder has been renamed to scales[id].border.display```
https://www.chartjs.org/docs/latest/migration/v4-migration.html

`options.scales[scaleId].border`
this display value is default = true
removed as not required
https://www.chartjs.org/docs/latest/axes/styling.html#border-configuration